junction/conferences/management/commands/sample_data.py

```diff
@@ -17,14 +17,16 @@
 # Junction Stuff
 from junction.base import constants
 from junction.conferences.models import Conference
-from junction.proposals.models import ProposalSection, ProposalType
+from junction.proposals.models import (ProposalSection, ProposalType,
+                                       Proposal, ProposalComment)
 
 NUM_USERS = getattr(settings, "SAMPLE_DATA_NUM_USERS", 10)
 NUM_CONFERENCES = getattr(settings, "SAMPLE_DATA_NUM_CONFERENCES", 4)
-NUM_EMPTY_CONFERENCES = getattr(settings, "SAMPLE_DATA_NUM_EMPTY_CONFERENCES",
-                                2)
+NUM_EMPTY_CONFERENCES = getattr(settings, "SAMPLE_DATA_NUM_EMPTY_CONFERENCES", 2)
 NUM_PROPOSAL_SECTIONS = getattr(settings, "NUM_PROPOSAL_SECTIONS", 5)
 NUM_PROPOSAL_TYPES = getattr(settings, "NUM_PROPOSAL_TYPES", 8)
+NUM_PROPOSALS = getattr(settings, "NUM_PROPOSAL", 10)
```

Comment:
Can we have proposal of different type like `draft, deleted` ? These can be separate variables.